specification/cognitiveservices/Language.AnalyzeConversations-authoring/models/common.tsp

```diff
@@ -31,7 +31,7 @@ union ProjectKind {
   /**
    * A project to build conversation summarization models which are able to summarize long conversations.
    */
-  @added(Versions.v2023_04_15_preview)
+  @added(Versions.v2024_11_15_preview)
```

Comment:
There is no more custom summarization conversation

Reply:
Will remove them separately, donnt want to trigger breaking changes yet.